web/template/partial/stream/bookmarks.gohtml

```diff
@@ -0,0 +1,153 @@
+{{/* 1st parameter: 'streamID' */}}
+{{define "bookmarks-modal"}}
+    <div x-data="{showAddMenu: false}"
+         class="relative h-full border rounded-lg dark:border-gray-600">
```

Comment:
I think the modal looks great! However, when I am viewing it on mobile, it takes quite a large amount of space. I think it would be better to collapse it, e.g. adjusting the height to the height used by the bookmarks.
<img width="294" alt="grafik" src="https://user-images.githubusercontent.com/30267166/205437854-b7aa0501-fc3b-4473-bebc-14079f11263b.png">

Reply:
As I will work on the mobile version as a whole soon, I think the best solution for now is a dialog:
<img width="320" alt="image" src="https://user-images.githubusercontent.com/29633518/205561110-a1749bb1-3c2b-4bec-b75c-3c57b23482f6.png">

Reply:
Looks great!